src/ast/ast_validations.c

```diff
@@ -442,6 +442,39 @@ static AST_Validation _Validate_MATCH_Clause_Filters(const cypher_astnode_t *cla
 	return AST_VALID;
 }
 
+static AST_Validation _ValidateNodeAlias(const cypher_astnode_t *node, TrieMap *identifiers,
```

Comment:
Consider renaming `identifiers` to reflect that we're dealing with a set of edge aliases.